pkg/urbit/daemon/main.c

```diff
@@ -991,6 +993,84 @@ _cw_intr_win(c3_c* han_c)
 }
 #endif
 
+c3_c 
+*_cw_eval_get_input(FILE* fp, size_t size)
```

Comment:
This should have a standard function comment, then  `c3_c*` on the next line, with rest of the signature on the line after.